Mathlib/LinearAlgebra/TensorProduct/Pi.lean

```diff
@@ -0,0 +1,86 @@
+/-
+Copyright (c) 2024 Judith Ludwig, Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Judith Ludwig, Christian Merten
+-/
+import Mathlib.LinearAlgebra.TensorProduct.Tower
+import Mathlib.LinearAlgebra.Pi
+
+/-!
+
+# Tensor product and products
```

Comment:
Can you mention somewhere also the file `Mathlib.LinearAlgebra.TensorProduct.Prod`? (And maybe also the other way, mentioning this file there).